File: HelloWorldScala/test/src/PlotSpec.scala
/* cspell: disable-next-line */
package HelloWorld

// cSpell:ignore splotly, munit

/**
 * ./mill mill.scalalib.GenIdea/idea
 *
 * ./mill -i HelloWorld.test
 * ./mill -i HelloWorld.test.testLocal
 * ./mill -i HelloWorld.test HelloWorld.PlotSpec.*
 * ./mill -i HelloWorld.test HelloWorld.PlotSpec.hello // test("hello")
 *
 * Extending `TestCase` to get access to `setUp`
 *
 * https://github.com/sbt/junit-interface#junit-interface
 * ./mill -i managed.test --tests=test1           Only matches test names (not classes)
 *
 *
 */
class PlotSpec extends munit.FunSuite {

  test("hello") {
    val obtained = 42
    val expected = 43
    assertEquals(obtained, expected)
  }

}

File: HelloWorldJava/jtest/src/helloworld/PlotSpec.java
/* cspell: disable-next-line */
package helloworld;

// cSpell:ignore splotly, munit


import static org.junit.jupiter.api.Assertions.assertEquals;
import org.junit.jupiter.api.Test;

/**
 * ./mill mill.scalalib.GenIdea/idea
 *
 * ./mill -i HelloWorldJava.jtest
 * ./mill -i HelloWorldJava.jtest.testLocal
 * ./mill -i HelloWorldJava.jtest --tests=hello.*
 * ./mill -i HelloWorldJava.jtest --tests=he.*o.*
 *
 * Extending `TestCase` to get access to `setUp`
 *
 * https://github.com/sbt/junit-interface#junit-interface
 * https://github.com/sbt/sbt-jupiter-interface
 * ./mill -i managed.test --tests=test1           Only matches test names (not classes)
 *
 *
 */
public class PlotSpec {

  @Test
  public void hello() {
    assertEquals(2, 1);
  }

}

